ansible/roles/mailhog/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Download MailHog
+  get_url:
+    url: "https://github.com/mailhog/MailHog/releases/download/v1.0.0/MailHog_linux_amd64"
+    dest: /usr/local/bin/mailhog
+    owner: root
+    group: root
+    mode: 0755
+  become: true
+
+- name: Add MailHog basic authentication file
+  copy:
+    content: |
+      {{ mailhog_username }}:{{ mailhog_password_bcrypt }}
```

Comment:
If these are empty, is basic auth disabled by MailHog? We want it to be disabled on Vagrant.